src/twisted/enterprise/adbapi.py

```diff
@@ -290,7 +290,7 @@ def _runWithConnection(self, func, *args, **kw):
                 conn.rollback()
             except BaseException:
                 log.err(None, "Rollback failed")
-            compat.reraise(excValue, excTraceback)
+            raise excValue.with_traceback(excTraceback)
```

Comment:
```suggestion
            raise
```

Reply:
pretty sure `raise` takes the __traceback__ from the correct place here anyway...